openapi/external.yaml

```diff
@@ -1,53 +1,491 @@
+              schema:
+                $ref: '#/components/responses/BadRequest'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/Unauthorized'
+        '403':
+          description: Forbidden
+        '409':
+          description: Conflict
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/InternalServerError'
+        '502':
+          description: Bad Gateway
+        '504':
+          description: Gateway Timeout
+  /logout:
+    post:
+      operationId: logoutFromIOApp
+      summary: LogoutFromIOApp
+      description: Logout User from IO App
+      security:
+        - bearerAuth: []
+      responses:
+        '200':
+          description: Success
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/BadRequest'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/Unauthorized'
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/InternalServerError'
+        '502':
+          description: Bad Gateway
+        '504':
+          description: Gateway Timeout
+  /sessions:
+    get:
+      operationId: getSessionsList
+      summary: GetSessionsList
+      description: Retrieve a user active sessions list
+      security:
+        - bearerAuth: []
+      responses:
+        '200':
+          description: Success
+        '404':
+          description: Not Found
+  /profile:
+    get:
+      operationId: getProfile
+      summary: GetProfile
+      description: Retrieve a user profile
+      security:
+        - bearerAuth: []
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProfileData'
+        '400':
+          description: Invalid request.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/BadRequest'
+        '401':
+          description: Unauthorized
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemJson'
+        '429':
+          description: Too Many Requests
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/InternalServerError'
+  /session-state:
+    post:
```

Comment:
There is no need to have a post operation here. We need it in fn-fast-login because we need to pass the fiscal code in body (privacy policy to avoid logging sensitive data)